src/operator/tensor/cast_storage-inl.cuh

```diff
@@ -14,13 +14,288 @@
 namespace mxnet {
 namespace op {
 using mshadow::cuda::kBaseThreadNum;
+using mshadow::Shape1;
```

Comment:
Keep the `using namespace` scope local, especially not using them in a header file.